webapp/channels/src/components/admin_console/team_channel_settings/convert_confirm_modal.tsx

```diff
@@ -101,9 +113,14 @@ const messages = defineMessages({
     toPrivateMessage: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPrivateMessage',
 
-        // This eslint-disable comment can be removed once this component no longer uses FormattedMarkdownMessage
         // eslint-disable-next-line formatjs/no-multiple-whitespaces
-        defaultMessage: 'When you convert **{displayName}** to a private channel, history and membership are preserved. Publicly shared files remain accessible to anyone with the link. Membership in a private channel is by invitation only.  \n \nAre you sure you want to convert **{displayName}** to a private channel?',
+        defaultMessage: 'When you convert {displayName} to a private channel, history and membership are preserved. Publicly shared files remain accessible to anyone with the link.',
+    },
+    toPrivateMessageConfirmation: {
+        id: 'admin.team_channel_settings.convertConfirmModal.toPrivateMessageConfirmation',
+
+        // eslint-disable-next-line formatjs/no-multiple-whitespaces
+        defaultMessage: 'Are you sure you want to convert {displayName} to a private channel?',
     },
     toPrivateTitle: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPrivateTitle',
```

Comment:
similarly here as well this message is now changed then but the en.json isnt updated with this change

Reply:
Done!